package.json

```diff
@@ -1,11 +1,15 @@
 {
   "name": "react-iiif-media-player",
-  "version": "1.0.8",
-  "description":
-    "react-iiif-media-player React component for rendering a media player for a IIIF 3.0 spec manifest",
+  "version": "1.1.0",
```

Comment:
I see latest in npm is `1.1.2`.  Should this be bumped to `1.2.0` or does the npm build handle this version automatically?

Reply:
Yea, every time you do a `yarn publish`, you have to specify an updated version.  Once we get the NPM registry version stable, I'll do another PR to make sure `master` here is in sync.

Or we go the route of pulling in from a branch in this repo, specify that in a `package.json` file in `avalon-bundle`, but then wondering how we'd keep the branch, and the registry package in sync...